docs/en/05-supported-backends/rknn.md

```diff
@@ -0,0 +1,81 @@
+# RKNN support
+
+This tutorial is based on Linux systems like Ubuntu-18.04 and Rockchip NPU like `rk3588`.
+
+## Installation
+
+It is recommended to create a virtual environment for the project.
+
+1. get RKNN-Toolkit2 through:
+
+   ```
+   git clone https://github.com/rockchip-linux/rknn-toolkit2
+   ```
+
+2. install RKNN python package following [official doc](https://github.com/rockchip-linux/rknn-toolkit2/tree/master/doc).
+
+3. reinstall MMDeploy from source following the [instructions](../01-how-to-build/build_from_source.md). Note that there are conflicts between the pip dependencies of MMDeploy and RKNN. Here is the suggested packages versions for python 3.6:
+
+   ```
+   protobuf==3.19.4
+   onnx==1.8.0
+   onnxruntime==1.8.0
+   torch==1.8.0
+   torchvision==0.9.0
+   ```
+
+To work with models from [MMDetection](https://github.com/open-mmlab/mmdetection/blob/master/docs/get_started.md), you may need to install it additionally.
+
+## Usage
+
+Example:
+
+```bash
+python tools/deploy.py \
+    configs/mmdet/detection/detection_rknn_static.py \
+    /mmdetection_dir/mmdetection/configs/yolo/yolov3_d53_mstrain-608_273e_coco.py \
+    /tmp/snapshots/yolov3_d53_mstrain-608_273e_coco_20210518_115020-a2c3acb8.pth \
+    tests/data/tiger.jpeg \
+    --work-dir ../deploy_result \
```

Comment:
Please add `--dump-info`

Reply:
But we did not support SDK stuff here.

Reply:
--dump-info is the precondition. Without it, how can SDK do testing?
It's not SDK's responsibility to get the deployment meta info because it has no idea how model converter does the postprocessing.